### ISSUE TITLE
Run FBA tests on all BiGG models

### ISSUE THREAD

**Issue by shalinshah1993** [draeger-lab/SBSCL#4]
_From @matthiaskoenig on June 11, 2018 13:2_

FBA simulations with all BiGG models should be performed and compared against reference solutions from cobrapy. 
BiGG models are not part of the repository, but must be downloaded for testing in CI or locally. Access is via environment variable.

_Copied from original issue: shalinshah1993/SBSCL#28_

**Comment by shalinshah1993** [comment]
_From @matthiaskoenig on June 11, 2018 13:7_

Most of this is implemented in
https://github.com/shalinshah1993/SBSCL/commit/cefc069f6000ffe7ff252a6049413a5f422d33e2

This issue depends on #24, i.e. only if FBA is working than it can be run on all BiGG models.

**Comment by hemilpanchiwala** [comment]
@matthiaskoenig, @shalinshah1993 
Now, all FBA tests of BiGG models are running and passing successfully, so should I link this issue to PR #54 so that it can be closed on PR merge?

Commits Link: https://github.com/draeger-lab/SBSCL/commit/c40ef21edde633cdff534134b671137d06eb6eca https://github.com/draeger-lab/SBSCL/commit/9b924969c3c7cb42e7d5de27ae2f025034697222

**Comment by matthiaskoenig** [comment]
Yes, link the issue
